chat-plugins/github.js

```diff
@@ -0,0 +1,93 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
```

Comment:
Blah, I'm not sure how I feel about this variable name.  What about ``gitConfig`` instead?